lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,74 @@
+require 'httparty'
+
+class EdamamApiWrapper
+  BASE_URL = "https://api.edamam.com/"
+  ID = ENV["APP_ID"]
+  KEY = ENV["APP_KEY"]
+
+
+  def self.find_recipes_for(query)
+    url = BASE_URL + "search?q=#{query}" + "&app_id=#{ID}" + "&app_key=#{KEY}" + "&from=0" + "&to=50"
+
+    encoded_uri = URI.encode(url)
+
+    data = HTTParty.get(encoded_uri)
+    recipe_list = []
+
+    if data["hits"]
+      data["hits"].each do |recipe_data|
+        recipe_list << create_recipe(recipe_data)
+      end
+    end
+
+    return recipe_list
+  end
+
+  def self.find_recipe(parsed_id)
+
+    url = BASE_URL + "search?r=" + "http%3A%2F%2Fwww.edamam.com%2Fontologies%2Fedamam.owl%23" + "#{parsed_id}" + "&app_id=#{ID}" + "&app_key=#{KEY}"
+
+    data = HTTParty.get(url)
+
+    if data.first
+      return [details_recipe(data.first)]
+    else
+      return []
+    end
+
+  end
+
+  private
+
+  def self.create_recipe(api_params)
+    return Recipe.new(
```

Comment:
I like that you've broken this out as a separate method and made it `private` - good organization.